CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 ## 🔧 Tech
 * Use `<SharingBannerPlugin />` and `useSharingInfos()` from `cozy-sharing` instead of internal components
 * Fixed an error in Search result when the result contained at least one Cozy Note
+* Update cordova to 8.1.2 and cordova-android to 9.1.0
```

Comment:
this is not the goal of this upgrade. The goal is to be able to upload multiple files at once on android. We should add it in the changelog.